lib/msal-browser/docs/access-token-proof-of-possession.md

```diff
@@ -151,6 +151,14 @@ The Proof-of-Possession authentication scheme relies on an asymmetric cryptograp
 
 In the event of refreshing a bound access token, MSAL will delete the cryptographic keypair that was generated when requesting the expired bound access token, generate a new cryptographic keypair for the new access token, and store the new keypair in the keystore.
 
+## Advanced feature: Application managed cryptographic keypair
+
+> :warning: We do not recommend using this feature unless you are familiar with the [Proof of Possession protocol](https://oauth.net/2/dpop/) and have a specific requirement to generate your own cryptographic keypair. For most cases, we recommend the PoP usage as described in the rest of this document.
+
+If you choose to generate your own cryptographic keypair, then this feature enables the application to provide the `reqCnf` as a request parameter. MSAL JS ensures the token issuer embeds the `cnf` in the token but returns the issued token _unsigned_. The onus of signing the access token before it is forwarded to the intended resource will be on the application. 
```

Comment:
This needs to be updated to reflect the changes we made since this was written & would be good to include some sample code. Doesn't need to block the release but should be a fast follow if you need more time.

Reply:
I changed the reqCnf to popKid.. I will add more documentation in subsequent PRs after the release.